iaac/terraform/aws-infra/cognito/custom_domain.tf

```diff
@@ -18,13 +26,6 @@ resource "aws_route53_record" "pre_cognito_domain_a_record" {
   }
 }
 
-# Cognito requires a certificate in N.Virginia in order to have a custom domain for a user pool
-# https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-add-custom-domain.html
-provider "aws" {
```

Comment:
revert?

also lets move this to a separate file called `providers.tf`

Reply:
doesn't work with this here:
- conflict when passing in the provider
- makes it a legacy module

Reply:
ok for now